docs/js/version-select.js

```diff
@@ -0,0 +1,63 @@
+window.addEventListener("DOMContentLoaded", function () {
+  // This is a bit hacky. Figure out the base URL from a known CSS file the
+  // template refers to...
+  var ex = new RegExp("/?assets/fonts/material-icons.css$");
```

Comment:
Where does `material-icons` como from? Cannot find it mentioned anywhere

Reply:
this is generated when I run the `mike install-extras` 

https://github.com/jimporter/mike#building-your-docs

Reply:
I still edited this file to add the classes to do the <select> more "material like" .. by default it creates the ugly default <select>

Reply:
Is this working on this repository? When I tried this on the codacy/docs repository, it couldn't find the `material-icons.css` file and the whole JavaScript would fail.

I completely rewrote this `version-select.js` file to make it way less hacky, but I still need to solve the problem of the selected version not being kept between page reloads:

https://github.com/codacy/docs/pull/31/files